src/components/CountryDetail.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
- [ ] There are a lot of linters that have been disabled in different files. As the previous reviewer suggested Kindly fix them so your code respect best practices and clean code

Reply:
Hello @Ismailco you have reviewed the old commit please kindly look at the files changed you will i have updated the layout and also removed the majority of those linters!

Reply:
@Pamphilemkp, it's the last commit, you can see in the file there is `eslint-disable`, kindly fix them before requesting a new review